examples/example_gene_fusion_class.json

```
@@ -1,103 +1,84 @@
 {
     "r_frame_preserved": true,
-    "regulatory_elements": [
-        {
-            "type": "promoter",
-            "value_id": "hgnc:100",
-            "label": "ASIC1"
-        },
-        {
-            "type": "enhancer",
-            "value_id": "hgnc:200",
-            "label": "G1"
-        }
-    ],
     "protein_domains": [
         {
-            "status": "preserved",
-            "name": "tyrosine kinase catalytic domain",
-            "id": "interpro:IPR020635",
+            "status": "lost",
+            "name": "cystatin domain",
+            "id": "interpro:IPR000010",
             "gene": {
-                "value_id": "hgnc:8031",
-                "label": "NTRK1",
-                "type": "GeneDescriptor"
+                "id": "gene:CST1",
+                "value_id": "hgnc:2743",
+                "label": "CST1"
```

Do we want to add in type? I feel like we're inconsistent when adding type for this project so I'm unsure when we are supposed to. 
```suggestion
                "label": "CST1"
                "type": "GeneDescriptor"
```

---

yes -- copy/paste error on my part